crates/cairo-lint-core/tests/test_files/manual_unwrap_or_default/manual_unwrap_or_default

```diff
@@ -0,0 +1,385 @@
+//! > manual unwrap or default for if let with default
+
+//! > cairo_code
```

Comment:
add a test where the lint shouldn't be triggered

Reply:
Done